cdist/conf/type/__ufw/man.rst

```diff
@@ -0,0 +1,46 @@
+cdist-type__ufw(7)
+================
+
+NAME
+----
+cdist-type__ufw - Install the Uncomplicated FireWall
+
+
+DESCRIPTION
+-----------
+Installs the Uncomplicated FireWall. Most modern distributions carry UFW in their main repositories, but on CentOS this type will automatically enable the EPEL repository.
+
+Some global configuration can also be set with this type.
+
+OPTIONAL PARAMETERS
+-------------------
+state
+    Either "enabled", "running", "present", or "absent". Defaults to "enabled", which registers UFW to start on boot.
+
+logging
+    Either "off", "low", "medium", "high", or "full". Will be passed to `ufw logging`. If not specified, logging level is not modified.
+    
+
+EXAMPLES
+--------
+
+.. code-block:: sh
+
+    # Install UFW
+    __ufw
+
+
+SEE ALSO
+--------
+:strong:`ufw`\ (8)
+
+
+AUTHORS
+-------
+Mark Polyakov <mark@markasoftware.com>
+
+
+COPYING
+-------
+Copyright \(C) 2019 Mark Polyakov. Free use of this software is
+granted under the terms of the GNU General Public License version 3 (GPLv3).
```

Comment:
Can you please use `COPYING` the same as other official types, if you agree? See e.g. `__file` type
(`either version 3 of the License, or (at your option) any later version.`).
Thanks!

Reply:
Oops, used @asteven 's script to create the type boilerplate and it had this in it!